.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,146 @@
+name: Build and test wheels
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - release-*
+      - "*wheel*" # must quote since "*" is a YAML reserved character; we want a string
+    tags:
+      - "*"
+  pull_request:
+    branches:
+      - "*wheel*" # must quote since "*" is a YAML reserved character; we want a string
+
+jobs:
+  build_wheels:
+    name: Wheel ${{ matrix.buildplat[0] }}-${{ matrix.buildplat[1] }}-${{ matrix.python }}
+    runs-on: ${{ matrix.buildplat[0] }}
+    strategy:
+      matrix:
+        buildplat:
+          - [ubuntu-22.04, manylinux_x86_64]
+          - [macos-13, macosx_x86_64]
+          - [macos-14, macosx_arm64]
+          - [windows-2022, win_amd64]
+        python: ["cp39", "cp310", "cp311", "cp312"]
```

Comment:
Do we have a problem with cp38? Or just considered deprecated?

Reply:
I forgot why, I can try it out if you want

Reply:
https://github.com/dudoslav/TileDB-Py/actions/runs/9580398914